src/components/Checkbox/PCheckbox.vue

```diff
@@ -97,8 +97,8 @@
   focus:ring-offset-focus-ring
   focus:ring-offset-focus-ring-offset;
   background-color: var(--p-color-input-bg);
```

Comment:
Open to reverting, but something I noticed whilst digging through this:
these `color` rules don't do anything since the "check"(:checked) and the "minus"(:indeterminate) are background-image SVGs from the tailwind forms plugin. 

We could certainly _make_ these work with some more tweaks, but atm they don't. 

Additionally, the tailwindcss-forms base styles are designed to support setting the background color via utilities like `text-blue-500` so the different pseudo selectors are styled like `[type='checkbox']:checked { background-color: currentColor }` which means that for each state, we don't have to override the background-color. Instead, we can just set the color (similar to using a `text-blue-500` as is in their docs). 

So here, I'm removing the background-color overrides in favor of just setting via `color`. I think this also maps more closely to newer checkbox styling via `accent-color`. 


That said, I could see that it might be a little confusing this way that `color` is what's ultimately influencing the "background-color"...